src/backend/expungeservice/Dockerfile.web

```diff
@@ -1,10 +1,10 @@
 FROM recordexpungpdx/recordexpungpdx:expungeservicebase
 
 RUN mkdir -p /var/www/expungeservice
+RUN mkdir /var/www/logs/
```

Comment:
Regarding the log files that are being created:

@wittejm It seems we’re capturing some logs that are being stored in a log file. If so let’s look into two things for the purpose of ensuring our log file stays a reasonable size (for cost and speed reasons). Perhaps you’ve already thought of this and/or have implemented an approach.

1. we create logs files for each day. And every month or two months we delete the oldest files and keep the most recent. 
Or,
2. Perhaps Heroku has a logging file that we can send the log data to that is automatically kept a certain size. Therefore eliminating the need to create our own logging directory.

Reply:
These are good ideas. I'll make an issue for improving our logging approach